thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/jdbc/GroupedAnomalyResultsManagerImpl.java

```diff
@@ -0,0 +1,132 @@
+package com.linkedin.thirdeye.datalayer.bao.jdbc;
+
+import com.linkedin.thirdeye.client.DAORegistry;
+import com.linkedin.thirdeye.datalayer.bao.GroupedAnomalyResultsManager;
+import com.linkedin.thirdeye.datalayer.bao.MergedAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.dto.GroupedAnomalyResultsDTO;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import com.linkedin.thirdeye.datalayer.pojo.GroupedAnomalyResultsBean;
+import com.linkedin.thirdeye.datalayer.pojo.MergedAnomalyResultBean;
+import com.linkedin.thirdeye.datalayer.util.Predicate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import org.apache.commons.collections.CollectionUtils;
+import org.modelmapper.ModelMapper;
+
+public class GroupedAnomalyResultsManagerImpl extends AbstractManagerImpl<GroupedAnomalyResultsDTO>
+    implements GroupedAnomalyResultsManager {
+
+  protected static final ModelMapper MODEL_MAPPER = new ModelMapper();
+
+  protected GroupedAnomalyResultsManagerImpl() {
+    super(GroupedAnomalyResultsDTO.class, GroupedAnomalyResultsBean.class);
+  }
+
+  @Override
+  public Long save(GroupedAnomalyResultsDTO groupedAnomalyResultDTO) {
```

Comment:
The super method could not handle nested DTO objects. Since a grouped anomaly contains multiple merged anomalies, the save and update methods of grouped anomaly manager need to handle the list of merged anomaly dtos.